pkg/rehearse/metrics.go

```diff
@@ -96,8 +104,22 @@ func (m *Metrics) RecordOpportunity(toRehearse config.Presubmits, reason string)
 	}
 }
 
-func (m *Metrics) RecordActual(rehearsals []*prowconfig.Presubmit) {
-	for _, job := range rehearsals {
+func (m *Metrics) RecordPeriodicsOpportunity(periodics []prowconfig.Periodic, reason string) {
+	for _, job := range periodics {
+		if _, ok := m.Opportunities[job.Name]; !ok {
+			m.Opportunities[job.Name] = []string{reason}
+		} else {
+			m.Opportunities[job.Name] = append(m.Opportunities[job.Name], reason)
+		}
+	}
+}
+
+func (m *Metrics) RecordActual(presubmits []*prowconfig.Presubmit, periodics []prowconfig.Periodic) {
```

Comment:
~~Should this method also be split to two, just like opportunities were?~~
Scratch that, I now see how this is used in `main` and keeping it as one makes sense.